lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -22,7 +23,7 @@ export interface CdnSiteHostingConstructProps
  */
 export class CdnSiteHostingConstruct extends cdk.Construct {
   public readonly s3Bucket: s3.Bucket;
-  public readonly cloudfrontWebDistribution: cloudfront.CloudFrontWebDistribution;
+  public readonly cloudfrontDistribution: cloudfront.Distribution;
```

Comment:
Have we checked the projects to see if we're relying on this in some way?

Reply:
I haven't but as I was intending a breaking change to the project I thought it wouldn't matter?

Reply:
It might just help highlight anything where we're relying on this for something that will no longer be available. In all likelihood, we won't be relying upon it in this way, but if we were doing that and several of the apps were then blocked from upgrading, the construct's usefulness would substantially lower.

Might be worth emailing round the devs to ask anyone who is using this to take a look at their own project and feed back (by a short deadline). I'll look at user-utils, for instance.